contrib/mesos/pkg/controllermanager/controllermanager.go

```diff
@@ -305,6 +306,17 @@ func clientForUserAgentOrDie(config client.Config, userAgent string) *client.Cli
 	return kubeClient
 }
 
+// clientsetForUserAgentOrDie is going to replace clientForUserAgentOrDie
+func clientsetForUserAgentOrDie(config client.Config, userAgent string) release_1_1.Interface {
```

Comment:
It seems this mesos directory is duplicating other functions as well. I don't want to make much effort to improve this directory. What do you say?